integration_tests/androidx_test/src/sharedTest/java/org/robolectric/integrationtests/axt/EspressoTest.java

```diff
@@ -23,15 +23,13 @@
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 import androidx.test.filters.SdkSuppress;
 import java.util.concurrent.atomic.AtomicReference;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.annotation.Config;
 import org.robolectric.integration.axt.R;
 
 /** Simple tests to verify espresso APIs can be used on both Robolectric and device. */
-@Ignore
```

Comment:
I remember @hoisie has fixed it, and related PR has been merged into master branch. @Umesh-01 could you rebase your PR to latest master branch? Thanks.

Reply:
I tried it two times, but when I push my code to remote repository it shows "On-ignore Espresso Test" commit separately and says to sync/update my repository. What should I do @utzcoz ??

Reply:
@Umesh-01 you can try https://stackoverflow.com/questions/7929369/how-to-rebase-local-branch-onto-remote-master.

Reply:
@utzcoz I did it in same way. First, I pulled all the changes from remote repo and then squashed the commits into a single one using rebase.

Reply:
I think I had also included the "On-ignore Espresso Test" commit in squash, that's why it was there.